addons/calendar/static/src/views/fields/many2many_attendee.js

```diff
@@ -47,6 +50,9 @@ export class Many2ManyAttendee extends Many2ManyTagsAvatarField {
             if (noEmail) {
                 tag.noEmail = true;
             }
+            if (unavailablePartnerIds.find((partner) => (tag.resId == partner.resId))) {
```

Comment:
"partner" is colliding with the "partner" variable here above.

Sugg:
`if (unavailablePartnerIds.find((unavailablePartner) => (partner.id === unavailablePartner.resId))) {`